src/AuthModule/components/Login/Login.tsx

```diff
@@ -1,11 +1,128 @@
+};
+  
+// senD Data to Api
+const onSubmit = async (data: FormData) => {
+  setSpinner(true);
+
+  try {
+    const response = await axios.post('https://upskilling-egypt.com:3003/api/v1/Users/Login', data );
+  
+    showSuccessToast('Login successfully');
+    localStorage.setItem("token",response.data.token);
+    savLoginData();
+    navigate('/dashboard');
+    // console.log(response)
+  } catch (error ) {
+    showErrorToast("An error occurred with login..");
+  } finally {
+    setSpinner(false);
+  }
+};
 
-export default function Login() {
   return (
-    <div className="Auth-container vh-100  d-flex align-items-center justify-content-center">
-      <div>
-        <h2 className="text-white ">Loginnnnn</h2>
+  
+      <div className="Auth-container vh-100 row align-items-center justify-content-center overflow-auto gx-0 flex-nowrap ">
+        <div className="logo  col-md-5 text-center">
+          <img src={logo} alt="logo" className="mb-3" />
+        </div>
+    
+        <div className="login-container  col-md-5 rounded-4 px-5 py-5">
+          <p className="text-white">Welcome Back!</p>
+          <h3 className="text-warning mb-3">Log In</h3>
+          <form onSubmit={handleSubmit(onSubmit)}>
+            <div className="form-group">
+              <label htmlFor="exampleFormControlInput1 mb-3">Email</label>
+              <input
+                type="email"
+                className="form-control"
+                {...register("email", {
+                  required: "email is required",
+                  pattern: {
+                    value: /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/,
+                    message: "email is not valid ",
+                  },
+                })}
+                placeholder="email"
+              />
+            </div>
+            {errors.email && (
+              <div className="alert alert-danger ">{errors.email.message}</div>
+            )}
+            <div className="form-group for-visibilty-password-container">
+              <label htmlFor="exampleFormControlInput1">password</label>
+              <input
+                type={showPassword ? "text" : "password"}
+                className="form-control"
+                placeholder="password"
+                {...register("password", {
+                  required: "password is required ",
+                  pattern: {
+                    value:
+                      /^(?=.*\d)(?=.*[a-z])(?=.*[A-Z])(?=.*[a-zA-Z]).{8,}$/,
+                    message:
+                      "Password must contain at least 8 characters, including upper and lowercase letters, and numbers",
+                  },
+                })}
+              />
+                    <button
+                    className="btn btn-outline-secondary for-visibilty-password-button"
+                    type="button"
+                    onClick={togglePasswordVisibility}
+                  >
+                    <i className="fa-solid fa-eye"></i>
```

Comment:
We need to change this icon according to the showPassword state, eye-slash when the password is hidden and eye only when the password is visible.